config/environments/production.rb

```diff
@@ -1,9 +1,13 @@
+require "active_support/core_ext/integer/time"
+
 Rails.application.configure do
+  config.action_controller.allow_forgery_protection = false
   config.action_controller.perform_caching = true
   config.action_mailer.perform_caching = false
   config.active_record.dump_schema_after_migration = false
   config.active_storage.service = :local
   config.active_support.deprecation = :notify
+  config.active_support.report_deprecations = false
   config.assets.compile = false
   config.assets.css_compressor = :sass
   # config.assets.js_compressor = :uglifier
```

Comment:
It's not relevant to this task, but in general I'm wondering if we shouldn't enable js compressor in production. :thinking:

Reply:
It would probably be better. I didn't do anything with it since it was out of scope